src/bento/dashboardTabData.js

```diff
@@ -110,8 +110,7 @@ export const tabIndex = [
   },
 ];
 //NOTE: Change 'getParticipants' to 'searchParticipants' in the backend.
-export const DASHBOARD_QUERY_NEW = gql`
-query search(
+export const DASHBOARD_QUERY_NEW = gql`query search(
```

Comment:
@Senait7-ux Looks like "query search" words move a line above it. Can we maintain same syntax like previous version (One before this commit).